packages/documentation/src/stories/components/tooltip/tooltip.stories.ts

```diff
@@ -0,0 +1,77 @@
+import { Args, Meta, StoryObj } from '@storybook/web-components';
+import { BADGE } from '../../../../.storybook/constants';
+import { html } from 'lit';
+
+const meta: Meta = {
+  title: 'Components/Tooltip',
+  component: 'post-tooltip',
+  parameters: {
+    badges: [BADGE.NEEDS_REVISION],
+    controls: {
+      exclude: ['class'],
+    },
+  },
+  render,
+  args: {
+    placement: 'top',
+    background: 'bg-primary',
+    message: 'Hi there 👋',
+  },
+  argTypes: {
+    message: {
+      name: 'Message',
+      description: 'Tooltip content',
+      control: {
+        type: 'text',
+      },
+    },
+    background: {
+      name: 'Background',
+      defaultValue: 'bg-primary',
+      description:
+        'Background color of the tooltip. Choose the one that provides the best contrast in your scenario.',
+      control: {
+        type: 'radio',
+        labels: {
+          'bg-primary': 'Primary',
+          'bg-yellow': 'Post yellow',
+        },
+      },
+      options: ['bg-primary', 'bg-yellow'],
+    },
+    slot: {
+      name: 'Slot',
+      control: false,
+      description:
+        'The tooltip default slot accepts any <a href="https://developer.mozilla.org/en-US/docs/Glossary/Inline-level_content">inline content</a> like `<span>` or `<post-icon>`, but no interactive content like `<a>` or `<button>` and no block level content like `<div>` or `p`.',
+    },
+  },
+  // Provide some margin for the tooltip to fit comfortably inside the iFrame with overflow: hidden
+  decorators: [
+    story =>
+      html`
+        <div style="margin-block: 5rem;">${story()}</div>
+      `,
+  ],
```

Comment:
Removing this decorator would clearly show, that the tooltip is placed on top of the page content and does not take any space in the layout:
![image](https://github.com/swisspost/design-system/assets/9716662/4bd230dc-f6f8-4092-9b0a-1637a7d8d937)

Reply:
That does not work in Firefox and Safari currently, as they are not supporting the `popover` attribute yet and Storybook has a lot of `overflow: hidden` in their maze of divs surrounding a story.

Reply:
That's really sad, because it means, that our tooltip component will not work in several browsers in a real situation, where we do not have control of parent elements with the css prop `overflow: hidden`.
I could imagine using the fully supported `dialog` element here - instead of the `popover` attribute - and restyling it to our needs could be a good solution for the moment, as noone can really say, when FF, Safari and Samsung Internet will support the popover API.
wdyt?

Reply:
Moving all tooltips to the end of the document now, this makes it work in safari and firefox similar too.

Reply:
Oh man, I feel bad about that! Like I've ruined your great solution. 😢

On the other hand, I think it makes sense to create stable solutions for all supported browsers.
Still, I'm looking forward when we're able to implement the popover API here (hopefully soon).